code/modules/mob/living/silicon/ai/decentralized/expansion_card_holder.dm

```diff
@@ -77,10 +77,27 @@ GLOBAL_LIST_EMPTY(expansion_card_holders)
 
 /obj/machinery/ai/expansion_card_holder/update_icon()
 	cut_overlays()
+
+	if(installed_cards.len > 0) {
+		var/mutable_appearance/top_overlay = mutable_appearance(icon, "expansion_bus_top")
+		add_overlay(top_overlay)
+	}
+	if(installed_cards.len > 1) {
+		var/mutable_appearance/bottom_overlay = mutable_appearance(icon, "expansion_bus_bottom")
```

Comment:
just name them expansion_bus_1 and expansion_bus_2 and then just have a singular var/mutable_appearance/the_overlay = mutable_appearance(icon, "expansion_bus_[min(installed_cards.len, 2)]") behind an if(installed_cards.len)

Reply:
they're two different overlays with two different animations, so that wouldn't work